spec/components/search_spec.rb

```diff
@@ -364,7 +364,7 @@ def create_pm(users:, group: nil)
 
         # private only
         results = Search.execute('in:all cheese', guardian: Guardian.new(u1))
-        expect(results.posts).to contain_exactly(private_post1)
+        expect(results.posts).to contain_exactly(private_post2)
```

Comment:
`private_post1` and `private_post2` has the same ranking so we want to be more intentional here and order by `posts.id DESC`